spec/knockout-fast-foreach-spec.js

```diff
@@ -24,6 +24,7 @@ function setupSynchronousFrameAnimation () {
 }
 
 describe("applying bindings", function () {
+  FastForEach.DEBUG = true;
```

Comment:
Ok, sounds reasonable, I will change it. I needed some logging or so, can't remember already.